bootstrap/src/deq.ml

```diff
@@ -0,0 +1,320 @@
+open Rudiments
+
+type 'a t =
+  usize * 'a Stream.t * 'a Stream.t * usize * 'a Stream.t * 'a Stream.t
+
+let c = 2
```

Comment:
A brief comment here would be useful to make it clear this isn't just a sloppy magic number.  Something like:

```ocaml
(* Increment size for rebuilding; must be either 2 or 3. *)
```